package.json

```diff
@@ -17,6 +17,7 @@
     "format": "prettier --write \"./packages/*/*/{src,bin,test}/**/*.{js,json,md}\"",
     "link-all": "node scripts/link-all.js packages",
     "unlink-all": "node scripts/unlink-all.js packages",
+    "check": "flow check",
```

Comment:
Would usually expect checks to have an associated run script for running locally. Can remove if this is unwanted.